<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>846</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>SSI</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QComboBox" name="objType">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>180</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>171</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Data type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>630</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Run and Export</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>60</y>
      <width>481</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Input layers</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>SSH Layer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Population Layer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="objLyr">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#objLyr{border-image: url(:/res/ObjectLayer-01.png);}
QPushButton#objLyr:hover{border-image:url(:/res/ObjectLayer-02.png)}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="rptLyr">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#rptLyr{border-image: url(:/res/ReportingLayer-01.png);}
QPushButton#rptLyr:hover{border-image: url(:/res/ReportingLayer-02.png)}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="view_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>490</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>490</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QPushButton" name="splLyr">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#splLyr{border-image: url(:/res/SamplingLayer-01.png);}
QPushButton#splLyr:hover{border-image:url(:/res/SamplingLayer-02.png)}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="stfLyr">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#stfLyr{border-image: url(:/res/StratifiedLayer-01.png);}
QPushButton#stfLyr:hover{border-image: url(:/res/StratifiedLayer-02.png)}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Sampling Layer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>300</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Reporting Layer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="del_1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>230</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>490</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>490</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>rptLyr</zorder>
    <zorder>view_3</zorder>
    <zorder>view_4</zorder>
    <zorder>view_1</zorder>
    <zorder>view_2</zorder>
    <zorder>splLyr</zorder>
    <zorder>stfLyr</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>objLyr</zorder>
    <zorder>del_1</zorder>
    <zorder>del_2</zorder>
    <zorder>del_3</zorder>
    <zorder>del_4</zorder>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>686</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/sandwich.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>686</y>
      <width>441</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Please add this reference in your study when applying SSI: Wang, J.-F., Haining, R., Liu, T.-J., Li, L.-F., &amp; Jiang, C.-S. (2013). Sandwich Estimation for Multi-Unit Reporting on a Stratified Heterogeneous Surface. Environment and Planning A: Economy and Space, 45(10), 2515â€“2534. https://doi.org/10.1068/a44710</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>793</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Developers: Yue Lin, liny@lreis.ac.cn; Chengdong Xu, xucd@lreis.ac.cn; Jinfeng Wang, wangjf@lreis.ac.cn</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>objType</zorder>
   <zorder>label</zorder>
   <zorder>run</zorder>
   <zorder>graphicsView</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
